test/functional/services/test_subjects.js

```diff
@@ -37,6 +38,14 @@ export function TestSubjectsProvider({ getService }) {
       return await find.existsByDisplayedByCssSelector(testSubjSelector(selector), timeout);
     }
 
+    async existOrFail(selector, timeout = 1000) {
+      log.debug(`TestSubjects.existOrFail(${selector})`);
+      const doesExist = await find.existsByDisplayedByCssSelector(testSubjSelector(selector), timeout);
```

Comment:
why not just do `const doesExist = await this.exists(selector, timeout);`?

Reply:
Good point, fixed!